File: Java/389. Find the Difference.java
class Solution {
    public char findTheDifference(String s, String t) {
        int ord_diff = 0;
        for(int i=0; i<s.length(); i++){
            ord_diff -= (int)s.charAt(i);
            ord_diff += (int)t.charAt(i);
        }
        return (char)(ord_diff + (int)t.charAt(t.length()-1));
    }
}

File: Python3/389. Find the Difference.py
class Solution:
    def findTheDifference(self, s: str, t: str) -> str:
        ord_diff = 0
        a = ord('a')
        
        for i in range(len(s)):
            ord_diff -= (ord(s[i]) - a)
            ord_diff += ord(t[i]) - a
            
        return chr(ord_diff+ord(t[-1]))
        